rosbridge_server/scripts/rosbridge_websocket.py

```diff
@@ -143,13 +140,17 @@ def check_origin(self, origin):
     rospy.init_node("rosbridge_websocket")
     signal(SIGINT, SIG_DFL)
 
+    # Authentication options
+    authentication_method = rospy.get_param('~authentication_method', None)
```

Comment:
Also, I wonder if using a string value with multiple values is the right thing to do, or if we should instead use a 'list' type of parameter.